Src/AutoFixtureUnitTest/FixtureTest.cs

```diff
@@ -948,8 +948,8 @@ public void InjectCustomUpperLimitWillCauseSutToReturnNumbersInLimit()
             var result = sut.Create<DoublePropertyHolder<int, long>>();
             // Verify outcome
             Assert.True(
-                (result.Property1 >= lower && result.Property1 < upper) &&
-                (result.Property2 >= lower && result.Property2 < upper)
+                (result.Property1 >= lower && result.Property1 <= upper) &&
+                (result.Property2 >= lower && result.Property2 <= upper)

```

Comment:
Maybe I have the wrong understanding of the bug.  The bug states that if I set a lower and upper limit of 1 and 10 respectively.  The final returned value will always be 10.  I believe the test above would never assert result.Property1 or 2 as 10 and so the test would not fail.  When I implemented my fix this was the only test that failed because 10 may possibly be the first value returned.  It is my understanding that the test was not complete.  If the upper limit is not inclusive then my fix is not needed.

Please let me know if that makes sense.